coreos/justfile

```diff
@@ -17,22 +17,35 @@ image := if os == "scos" { scos_img } else { fcos_img }
 os_name := if os == "scos" { scos_os } else { fcos_os }
 label := if os == "scos" { scos_label } else { fcos_label }
 archive := os + ".ociarchive"
+platform := "qemu"
+kbc_image := "quay.io/afrosi_rh/kbs-client-image:latest"
+clevis_pin_trustee_image := "quay.io/confidential-clusters/clevis-pin-trustee"
 
 build:
-    sudo podman build --no-cache --build-arg BASE={{base}} --build-arg COM_COREOS_OSNAME={{label}} -t {{image}} -f Containerfile .
+    sudo podman build --no-cache --build-arg BASE={{base}} --build-arg COM_COREOS_OSNAME={{label}} --build-arg KBS_CLIENT_IMAGE={{kbc_image}} --build-arg CLEVIS_PIN_TRUSTEE_IMAGE={{clevis_pin_trustee_image}} -t {{image}} -f Containerfile .
 
 oci-archive:
     sudo skopeo copy containers-storage:{{image}} oci-archive:{{archive}}
 
-osbuild-qemu:
+osbuild:
     #!/bin/bash
     set -xeuo pipefail
 
+    SELINUX_STATUS=$(getenforce)
+
+    if [ "$SELINUX_STATUS" = "Enforcing" ]; then
+        sudo setenforce 0
+    fi
+
```

Comment:
I do understand the step but I would prefer that everyone disable it manually rather then in a script where you type once the password. You can simply print a warning message